File: java/com/example/myapplication/MainActivity.java
package com.example.myapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    Button logBtn;
    Button nologbn;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        logBtn = (Button)findViewById(R.id.logbtn);
        nologbn = (Button)findViewById(R.id.nologbn);

        logBtn.setOnClickListener(new View.OnClickListener(){

            @Override
            public void onClick(View v) {
                Intent intent = new Intent(getApplicationContext(),SecondActivity.class);
                startActivity(intent);

            }
        });


        nologbn.setOnClickListener(new View.OnClickListener(){

            @Override
            public void onClick(View v) {
                Intent intent = new Intent(getApplicationContext(),FourActivity.class);
                startActivity(intent);

            }
        });


    }
}

File: main/java/com/example/lotto/MainActivity.kt
package com.example.lotto

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        main_bt1.setOnClickListener {
            val intent = Intent(this, LottoSelection::class.java)
            startActivity(intent)
        }
        main_bt2.setOnClickListener {
            val intent = Intent(this, LottoSiteCrawlingActivity::class.java)
            startActivity(intent)
        }
    }
}